src/components/AccountLists/AccountLists.tsx

```diff
@@ -78,8 +78,8 @@ const AccountLists = ({ data }: Props): ReactElement => {
                 totalPledges,
                 currency,
               }) => {
-                const receivedPercentage = receivedPledges / monthlyGoal;
-                const totalPercentage = totalPledges / monthlyGoal;
+                const receivedPercentage = receivedPledges / (monthlyGoal ?? 0);
+                const totalPercentage = totalPledges / (monthlyGoal ?? 0);
```

Comment:
Would this be an error if it divides by zero?

Reply:
I think a better way to address this may be to answer the question:

What should the percentage show if there isn't a valid monthly goal set?

Reply:
Ya I didn't know what to do here. I was just trying to make the types happy. We should probably show a dash or "No Goal" or something. I did see the NaN handling and sorta thought that was good enough for now but sounds like  it's not. I'll look into it more.

Reply:
I pushed a commit to make it more explicit that the fallback value is invalid and used [`Number.isFinite`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite) which will also handle infinity.

I could do an explicit check beforehand that `monthlyGoal` is defined if that seems better.